ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPool.java

```diff
@@ -469,6 +469,10 @@ && strictReplayProtectionShouldBeEnforcedLocally(chainHeadBlockHeader)
       return ValidationResultAndAccount.invalid(
           TransactionInvalidReason.INVALID_TRANSACTION_FORMAT,
           "EIP-1559 transaction are not allowed yet");
+    } else if (transaction.getType().equals(TransactionType.BLOB)
```

Comment:
this is the change that fixed the invalid block proposals, right?

Reply:
exactly! I considered updating a transaction validator to handle it, but decided against. it's not an invalid tx, it's just in the wrong context.

Reply:
can we use transaction.getType().supportsBlob() here?